app/src/main/res/layout/fragment_add_property_sold_layout.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <TextView
+        android:id="@+id/add_property_sold_title"
+        style="@style/Widget.FragmentAddPropertySoldTitleTextView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@id/add_property_sold_checkbox"
+        app:layout_constraintBottom_toBottomOf="@id/add_property_sold_checkbox"/>
+
+    <com.google.android.material.checkbox.MaterialCheckBox
+        android:id="@+id/add_property_sold_checkbox"
+        style="@style/Widget.FragmentAddPropertySoldCheckBox"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintStart_toEndOf="@id/add_property_sold_title"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <EditText
+        android:visibility="gone"
+        android:id="@+id/add_property_sold_edit_text"
+        style="@style/Widget.FragmentAddPropertySoldEditText"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:autofillHints="@string/dd_mm_yyyy"
+        android:hint="@string/dd_mm_yyyy"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@id/add_property_sold_checkbox"
+        app:layout_constraintTop_toTopOf="parent"
```

Comment:
format

Reply:
if i format autofill hint and hint, create a warning on edit text, do i have to format anyway and ignore warning in layout?